packages/teams-js/src/public/tasks.ts

```diff
@@ -128,4 +136,45 @@ export namespace tasks {
     taskInfo.width = taskInfo.width ? taskInfo.width : TaskModuleDimension.Small;
     return taskInfo;
   }
+
+  /**
+   * @hidden
+   * Converts {@link TaskInfo} to {@link AdaptiveCardDialogInfo}
+   * @param taskInfo - TaskInfo object to convert
+   * @returns - converted AdaptiveCardDialogInfo
+   */
+  function getAdaptiveCardDialogInfoFromTaskInfo(taskInfo: TaskInfo): AdaptiveCardDialogInfo {
```

Comment:
these two functions should be exported from dialog.ts and used here instead of duplicated